client/lib/reader-lists-feeds/constants.js

```diff
@@ -0,0 +1,6 @@
+module.exports.action = {
```

Comment:
Ideally it's `export const NAME = $AN_IMMUTABLE_VALUE`. That way the const bit goes along with the export and the thing is effectively frozen. 

With `export const action = { NAME: 'NAME' }` I can then `import action from 'place'; action.NAME = 'something else';`